strongbox-storage/strongbox-storage-api/strongbox/etc/conf/cassandra.yaml

```diff
@@ -0,0 +1,189 @@
+# ------------------------
+# Generic configuration
+# ------------------------
+cluster_name: Test Cluster
+snapshot_before_compaction: false
+auto_snapshot: false
+
+## ------------------------
+## Security
+## ------------------------
+authenticator: PasswordAuthenticator
+authorizer: AllowAllAuthorizer
+permissions_validity_in_ms: 2000
+
+## ------------------------
+## Network
+## ------------------------
+## start_rpc - disable thrift - it's deprecated and should not be used!
+start_rpc: false
+listen_address: 127.0.0.1
+storage_port: 7010
+ssl_storage_port: 7011
+start_native_transport: true
+native_transport_port: 49142
+native_transport_max_threads: 256
+
+## Coordinator related
+read_request_timeout_in_ms: 5000
+range_request_timeout_in_ms: 10000
+write_request_timeout_in_ms: 2000
+cas_contention_timeout_in_ms: 1000
+truncate_request_timeout_in_ms: 60000
+request_timeout_in_ms: 10000
+cross_node_timeout: false
+
+## Addresses of hosts that are deemed contact points.
+## Cassandra nodes use this list of hosts to find each other and learn
+## the topology of the ring. You must change this if you are running
+## multiple nodes!
+## - seeds is actually a comma-delimited list of addresses.
+## Ex: "seeds: <ip1>,<ip2>,<ip3>"
+seed_provider:
+    - class_name: org.apache.cassandra.locator.SimpleSeedProvider
+      parameters:
+          - seeds: "127.0.0.1"
```

Comment:
Correct me, if I am wrong, but I believe the convention for YAML tabs was two spaces, right?

Reply:
this file should not be here, probably committed accidentally
romoved